ambry-store/src/main/java/com.github.ambry.store/StoreMessageReadSet.java

```diff
@@ -81,19 +74,35 @@ long getOffset() {
   }
 
   long getSize() {
-    return size;
+    return info.getSize();
   }
 
   long getExpiresAtMs() {
-    return expiresAtMs;
+    return info.getExpirationTimeInMs();
   }
 
   StoreKey getStoreKey() {
-    return storeKey;
+    return info.getStoreKey();
+  }
+
+  short getAccountId() {
```

Comment:
do we need functions for all of these? Someone who wants this can just get the `MessageInfo` and use its APIs?